apps/sertra/components/ServerList.tsx

```diff
@@ -64,24 +53,20 @@ export const ServerList: React.FC<PropsWithChildren<ServerListProps>> = ({
   }, [community]);
 
   return (
-    <div>
-      <div className={styles.header}>
-        <div className={styles.titleContainer}>
-          <div className={styles.pageTitle}>Servers</div>
-          <div className={styles.gameTitle}>{gameDisplayName}</div>
-        </div>
+    <>
```

Comment:
Is there a reason for omitting the header container? IMO it would be beneficial to wrap every segment/block in a container.

Reply:
White kind of benefits do you have in mind? Robustness? Maintainability? I considered the `div` to be unnecessary, in the same line of thought that we yeeted Chakra's spacer divs.

Reply:
Maintainability by containment and separation of clear blocks of content / components. Basically the same reason as we have a container for the two title elements right below - to contain a one context of content.

While the header at this time has only the title, the fact can (and probably will) change with the addition of another element, such as pagination or a button, depending on the page. They both belong in the same context / block of content, and should be contained accordingly, so a container would be needed eventually anyway.

Having a container makes it so that any layout specific heavy lifting, such as content width, can be done at the container level, leaving the children agnostic about what's going on around them. An example of this is the comment above regarding having the whole site wrapped in a container. It's all the same principle. In the context of this header, we actually want the margin-bottom to apply to the whole header, not just the title.